app/clinic/controllers/scheduleListController.js

```diff
@@ -1,15 +1,14 @@
 'use strict';
 
 angular.module('clinic')
-        .controller('ScheduleListController', ['$rootScope', '$scope', 'patientService', 
-            'commonService', 'cohortService', 'openmrsPatientMapper', 'observationsService', 'visitService',
-                    function ($rootScope, $scope, patientService, 
-                        commonService, cohortService, patientMapper, observationsService, visitService) {
+        .controller('ScheduleListController', ['$rootScope', '$scope', 
+            'commonService', 'cohortService', 'observationsService', 'visitService',
+                    function ($rootScope, $scope, commonService, cohortService, observationsService, visitService) {
 
             $scope.hasSchedules = false;
     
             function init() {
-                var cohortMarckedAndCheckedInByProvider = 'ba110338-2f12-4a36-b2df-4abd5ae7c23e';
+                var cohortMarckedAndCheckedInByProvider = '2276035c-4b15-49c7-be31-dd11f7ac4246';
```

Comment:
After importing the cohorts you shared into my local instance; and do the following:
mysql> select name, uuid from serialized_object where name like 'POC%'; I get the following results:
![uuids_poc](https://cloud.githubusercontent.com/assets/1763308/17552896/72e4f420-5f0b-11e6-8654-12bcc43f2185.png)

Seems the uuid used do NOT match those in the DB?